src/ingress.ts

```diff
@@ -40,6 +40,11 @@ export interface IngressProps extends base.ResourceProps {
    * extension, if the ingress controller fulfilling the ingress supports SNI.
    */
   readonly tls?: IngressTls[];
+
+  /**
+   * Ingress Class Name is an optional String property.
```

Comment:
Describe the meaning, not the technicality. 

That is, what does `ingressClassName` mean? We usually lean on the k8s docs for that.